blerry/blerry.be

```diff
@@ -732,6 +784,13 @@ class Blerry
   def load_user_config()  # based on persist module
     var f
     var val
+    blerry_helpers.download_config("blerry_configurl.txt")
+    if path.exists("blerry_config.json")
```

Comment:
Should just use the "not" of the condition instead of leaving a blank action and only doing something in the else case

Reply:
Agreed.